src/api/controllers/clinical/assessment/assessment.controller.ts

```diff
@@ -277,6 +281,80 @@ export class AssessmentController extends BaseController{
         }
     };
 
+    answerQuestionList = async (request: express.Request, response: express.Response): Promise<void> => {
+        try {
+            
+            await this.setContext('Assessment.AnswerQuestion', request, response);
+
+            const id: uuid = await this._validator.getParamUuid(request, 'id');
+            const questionIds = request.params.listQuestionId;
+            const questionIdList =  questionIds.split(',');
+            let question = null;
+            let answerResponse: AssessmentQuestionResponseDto = null;
+            const answerResponseList = [];
+
```

Comment:
Add check for ServeListNodeChildrenAtOnce to be true before you start answering questions.

Reply:
done